src/calendar/header/index.js

```diff
@@ -99,14 +100,18 @@ class CalendarHeader extends Component {
   }
 
   renderWeekDays = (weekDaysNames) => {
-    const {disabledDaysIndexes} = this.props;
+    const {disabledDaysIndexes, renderWeekDays} = this.props;
     return weekDaysNames.map((day, idx) => {
       const dayStyle = [this.style.dayHeader];
 
       if (_.includes(disabledDaysIndexes, idx)) {
         dayStyle.push(this.style.disabledDayHeader);
       }
 
+      if (renderWeekDays) {
```

Comment:
If you're rendering the weekDays Text component yourself, this should be at the top (before line #105). Since you're not using 'dayStyle' const there is no need to compute it.